modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -181,13 +183,33 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		new KBTemplateModifiedDateComparator();
 		new KBTemplateTitleComparator();
 		new KBTemplateUserNameComparator();
+		new LayoutClassedModelUsageModifiedDateComparator();
 		new LayoutComparator();
+		new LayoutCreateDateComparator();
+		new LayoutModifiedDateComparator();
+		new LayoutPageTemplateEntryCreateDateComparator();
```

Comment:
@Valmir-unicap Please fix sorting, `LayoutPageTemplateEntry*` should come later than `LayoutPageTemplateCollection*`.